app/views/code_reviews/index.html.erb

```diff
@@ -1,19 +1,17 @@
 <h1>
   Code Reviews
+  <% if can? :manage, CodeReview %>
+    <%= link_to '+ New Code Review', new_code_review_path, class: 'text-success new-link' %>
+  <% end %>
```

Comment:
I feel like this link should be outside the `<h1>`, but I don't know what it looks like on the page.